### PR TITLE
[FLINK-12817][docs]Correct the import in Processing function example.

### PR DESCRIPTION
**Pull request by wenhuitang**
## What is the purpose of the change

This pull request correct the import in Processing function example in process_function.md. 

## Brief change log

Let the example import KeyedProcessFunction instead of ProcessFunction.

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): ( no )
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: ( no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/dev/stream/operators/process_function.zh.md [hunk 1]
```diff
@@ -97,9 +97,7 @@ import org.apache.flink.api.common.state.ValueStateDescriptor;
 import org.apache.flink.api.java.tuple.Tuple;
 import org.apache.flink.api.java.tuple.Tuple2;
 import org.apache.flink.configuration.Configuration;
-import org.apache.flink.streaming.api.functions.ProcessFunction;
-import org.apache.flink.streaming.api.functions.ProcessFunction.Context;
```
Comment (zentol) [inline docs/dev/stream/operators/process_function.zh.md]: same as above

File: docs/dev/stream/operators/process_function.md [hunk 2]
```diff
@@ -97,9 +97,7 @@ import org.apache.flink.api.common.state.ValueStateDescriptor;
 import org.apache.flink.api.java.tuple.Tuple;
 import org.apache.flink.api.java.tuple.Tuple2;
 import org.apache.flink.configuration.Configuration;
-import org.apache.flink.streaming.api.functions.ProcessFunction;
-import org.apache.flink.streaming.api.functions.ProcessFunction.Context;
-import org.apache.flink.streaming.api.functions.ProcessFunction.OnTimerContext;
+import org.apache.flink.streaming.api.functions.KeyedProcessFunction;
```
[the remark this answers is not in the document]
Comment (wenhuitang) [reply docs/dev/stream/operators/process_function.md]: Thanks a lot, I have addressed.

**Comment by StephanEwen** [comment]
The Scala imports still have the same issue as Chesnay mentioned for the Java imports...

**Comment by wenhuitang** [comment]
> The Scala imports still have the same issue as Chesnay mentioned for the Java imports...

The Scala use `KeyedProcessFunction[Tuple, (String, String), (String, Long)]#OnTimerContext` and `KeyedProcessFunction[Tuple, (String, String), (String, Long)]#OnTimerContext`, it seems that it does not require imports for KeyedProcessFunction.Context and KeyedProcessFunction.OnTimerContext.

**Pull request merged**